Adding an Event Listener/app.js

```diff
@@ -0,0 +1,6 @@
+var warning = document.getElementById("warning");
+var button = document.getElementById("makeItRed");
```

Comment:
Please use `const` since the variable values were not reassigned.

Reply:
Thanks, @lycoangelo, This is now updated.